01_Intro/Guess the Number/Guess the Number/Program.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Guess_the_Number
+{
+    class Program
+    {
+        enum difficulty { Easy = 100, Normal = 500, Hardcore = 1000 }
+        static void Main(string[] args)
+        {
+            Random rnd = new Random();
+            int ret = GetDifficulty();
+            int cheat = -322;
+            int dif = 0;
+            string diff = "";
+            switch (ret)
```

Comment:
das switch case kommt in die Funktion ```GetDifficulty()```. Dies gibt dann auch kein int mehr zurück sondern ```difficulty```.


In jedem case steht dann ca. das:

```csharp
case 1:
   return difficulty.Easy;

```

Reply:
dann kannst du dir auch das ```break``` sparen da ```return``` die Funktion sofort beendet und somit auch das switch case